cloudping.py

```diff
@@ -27,6 +28,7 @@ def ping(event, context):
             allow_redirects=options['allow_redirects'],
             timeout=options['timeout'])
         response.raise_for_status()
+        response_time = response.elapsed.total_seconds()
```

Comment:
Should we set response_time in the except block as well?

Reply:
I don't see a reason to do that. If the request fails, the response time is not relevant.